### DISCUSSION TITLE
Removing Original Visual

### DISCUSSION THREAD

**Discussion by composition-cassidy** [otomad/OtomadHelper#33] [Ideas]
Hello! Do you know when you make a video and it puts a new track on top of the original one to add new clips? I have an idea where instead of keeping the original track, it gets rid of it to make the process easier.

**Comment by otomad** [comment]
In the upcoming version, I have a new feature idea: add clips to the existing track instead of adding new tracks.

**Comment by otomad** [comment]
But there is a problem. When multiple tracks are generated (MIDI includes polyphony), the extra tracks do not know how to handle.
